### ISSUE TITLE
AutoAnswer property in config ignored

### ISSUE THREAD

**Issue by matko238** [voiceip/tinyphone#35]
Hi,

for some reason, autoAnswer=false is ignored from config.json, see following incoming call log:

```
2021-02-06 22:22:40,866 INFO  - WS: {"displayName":"Matko","from":"100","incomming":true,"party":"\"Matko\" <sip:100@192.168.4.193>","state":"INCOMING","type":"CALL"}
2021-02-06 22:22:40,866 INFO  - WS: {"callerId":"100","displayName":"Matko","id":0,"party":"\"Matko\" <sip:100@192.168.4.193>","sid":"57844acb-2e0e-4ab0-8f1d-f4a56382c9c0","state":"INCOMING","type":"CALL"}
2021-02-06 22:22:40,891 INFO  - WS: {"callerId":"100","displayName":"Matko","id":0,"party":"\"Matko\" <sip:100@192.168.4.193>","sid":"57844acb-2e0e-4ab0-8f1d-f4a56382c9c0","state":"CONNECTING","type":"CALL"}
2021-02-06 22:22:40,892 INFO  - WS: {"callerId":"100","displayName":"Matko","id":0,"party":"\"Matko\" <sip:100@192.168.4.193>","sid":"57844acb-2e0e-4ab0-8f1d-f4a56382c9c0","state":"CONFIRMED","type":"CALL"}
```

In config, auto answer is disabled:
`"autoAnswer" : false,`

**Comment by kingster** [comment]
What is the version of tinyphone that you are running?

**Comment by matko238** [comment]
Latest, tinyphone_installer.36.0.0.79

**Comment by matko238** [comment]
Issue solved with 36.0.0.80 version, thank you!
